docs/setup/manual-setup.md

```diff
@@ -3,41 +3,31 @@
 ## Requirements on your server
 
 - Node.js 10.13 or up
-
-- Database (PostgreSQL, MySQL, MariaDB, SQLite, MSSQL). Must use charset `utf8`: this is typically the
-  default in PostgreSQL and SQLite, while in MySQL and MariaDB UTF-8 might need to be set with
-  `alter database <DBNAME> character set utf8 collate utf8_bin;`
+- Database (PostgreSQL, MySQL, MariaDB, SQLite, MSSQL)  
+  The database must use charset `utf8`. This is typically the default in PostgreSQL and SQLite.  
+  In MySQL and MariaDB UTF-8 might need to be set with `alter database <DBNAME> character set utf8 collate utf8_bin;`  
   Be aware of older MySQL and MariaDB versions which sometimes use shorter representations of UTF-8 than 4 bytes.
   This can break if symbols with more bytes are used.
   You can use `alter database <DBNAME> character set utf8mb4 COLLATE utf8mb4_unicode_ci` to be on the safe side.
-
-- npm (and its dependencies, [node-gyp](https://github.com/nodejs/node-gyp#installation))
-
-- yarn
-
+- Npm (and its dependencies, [node-gyp](https://github.com/nodejs/node-gyp#installation))
+- Yarn
```

Comment:
Why did you capitalize the software names here?

Reply:
Because all other list items are also capitalized :D